src/js/users/types.ts

```diff
@@ -1,42 +1,68 @@
 import { AdministratorRoles } from "../administration/types";
 import { GroupMinimal, Permissions } from "../groups/types";
 
+/* Business to consumer specific details */
 type UserB2C = {
+    /* The display name (optional) */
     display_name?: string;
+    /* The family name (optional) */
     family_name?: string;
+    /* The given name (optional) */
```

Comment:
```suggestion
    /* The given name  */
```
No need to add optional, should be obvious at a glance from the type itself. Same is true for other instances as well